core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml

```diff
@@ -1,16 +1,33 @@
 ---
-- name: Set master hostname variable
-  set_fact:
-    master_hostname: "{{ groups['kubernetes_master'] | first }}"
+- name: Set facts for "classic epiphany k8s"
+  when: k8s_as_cloud_service is defined and k8s_as_cloud_service is false
+  block:
+    - name: Set master hostname variable
+      set_fact:
+        master_hostname: "{{ groups['kubernetes_master'] | first }}"
 
-- name: Set api server address variable
-  set_fact:
-    api_server_address: "https://{{ master_hostname }}:6443"
+    - name: Set api server address variable
+      set_fact:
+        api_server_address: "https://{{ master_hostname }}:6443"
+
+    - name: Set Kubernetes credentials
```

Comment:
I don't think this block is needed. Currently we create this kubeconfig file once in kubernetes_master and then it's removed in postflight. No need for creating it again, it's already there. :thinking:

Reply:
True, this is it what I just removed. I had to rebase my repo since this changed was made a few hours after I forked repo.